README.md

```diff
@@ -145,6 +145,32 @@ pip install -e .
 pip install -e .[transformers]
 ```
 
+## Installation (ROCm)

```

Comment:
can we shorten this? it has lots of overlap with the above

Reply:
I have push new changes. Can you help me to take a look at it?